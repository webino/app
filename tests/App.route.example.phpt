<?php
/**
 * Webino™ (http://webino.sk)
 *
 * @noinspection PhpUnusedParameterInspection
 * @outputmatchfile App.route.example.expected
 * @interpreter php-cgi
 *
 * @link        https://github.com/webino/app
 * @copyright   Copyright (c) 2019 Webino, s.r.o. (http://webino.sk)
 * @author      Peter Bačinský <peter@bacinsky.sk>
 * @license     BSD-3-Clause
 */

namespace Webino;

use Tester\Environment;

Environment::setup();
chdir(__DIR__ . '/../examples');


$container = new InstanceContainer;

/** @var HttpRequest $request */
$request = $container->make(HttpRequest::class, HttpRequest::defaults([
    HttpRequest::URI => 'example'
]));


$core = new BaseCore;
$app = $core->bootstrap();

$app->on(ExampleRoute::class, function (ExampleRoute $route) {
    return 'Example';
});

$app->dispatch($request);
