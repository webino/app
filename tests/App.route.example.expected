%a%
Content-Type: text/plain; charset=UTF-8

Example
